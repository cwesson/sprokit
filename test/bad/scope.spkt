
var a : int32 = 12;
var b : int32 = 12;

func test(const con : int32, const v : bool) : int32 {
	var c : int32 = 12;
	if v == true {
		var d : int32 = 12;
		if con == 2 {
			c = 4;
			d = 6;
		}else{
			c = 13;
		}
	} else {
		c = 14;
		d = 15;
	}
	return 1;
}
