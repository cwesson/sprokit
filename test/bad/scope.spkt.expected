Error on line 17:
  Variable d not declared.
